README-develop.md

```diff
@@ -20,6 +15,28 @@ open http://$(docker-compose port nginx 80)
 
 ## Setup for development
 
+Add core-bundle to the project:
+```sh
+git clone git@github.com:aakb/kontrolgruppen-core-bundle.git bundles/core-bundle
+```
+
+Change configuration in composer.json to use the local copy of the core-bundle: (make sure that the version constraint of the package is set to the current branch your local clone of the core bundle)
+Just make sure you don't commit theese changes, as they only are for the local setup.
+```json
```

Comment:
I think we should show console commands for making these changes:

```sh
composer config repositories.kontrolgruppen/core-bundle path bundles/core-bundle
composer require 'kontrolgruppen/core-bundle:dev-develop'
```
(symlinking is the default if possible, cf. https://getcomposer.org/doc/05-repositories.md#path)

Maybe we should even remove the `json` snippet …